aquadoggo/src/db/errors.rs

```diff
@@ -17,6 +17,10 @@ pub enum SqlStoreError {
     /// Error returned from BlobStore.
     #[error(transparent)]
     BlobStoreError(#[from] BlobStoreError),
+
+    /// Error returned from `DocumentStore` methods.
+    #[error(transparent)]
+    DocumentStorage(#[from] DocumentStorageError),
```

Comment:
Ah funny, I realized that the errors always end on "Storage" why the rest of the APIs always talk about "Store". Not important, just wanted to mark that.

Reply:
Oh yeh :sweat_smile: we can make this more consistent in a follow-up PR if we like.